test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] [OPTIONAL] Even if the previous reviewer didn't mention that It is important to know that this file should not appear on Github. The reason why we add the name of the file in the `.gitignore` file is to restrict git from tracking them and pushing them on Github. I have seen that you have added it in gitignore but It is on GitHub probably because you have pushed it before mentioning it in the gitignore file. To correct it, You will need to delete it manually on Github